exovetter/tce.py

```diff
@@ -158,3 +160,73 @@ def validate(self):
                 raise TypeError(f"Special param {key} must be an astropy "
                                 "Quantity")
         return True
+
+    def to_json(self, filename):
+        """Write a json file with standard file name
+
+        Parameters
+        ----------
+        filename : string
+        Filename to write the json string.
+
+        Returns
+        -------
+        tce_json : json formatted string
+
+        """
+        tmp = {}
+        for key in self.keys():
+            if isinstance(self[key], u.Quantity):
+                tmp[key] = self[key].value
+                unit_key = f'{key}_unit'
+                tmp[unit_key] = str(self[key].unit)
+            else:
+                tmp[key] = self[key]
+
+        tce_json = json.dumps(tmp)
+
+        if filename is not None:
+            fobj = open(filename, 'w')
+            fobj.write(tce_json)
+            fobj.close()
```

Comment:
Context manager is safer, as it guarantees closing the file handler even if write errors out.
```suggestion
            with open(filename, 'w') as fobj:
                fobj.write(tce_json)
```